mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMailboxMessageSearchIndexTest.java

```diff
@@ -170,7 +170,7 @@ void setUp() throws Exception {
             .internalDate(cal2.getTime())
             .body("My Otherbody2".getBytes(StandardCharsets.UTF_8))
```

Comment:
Should we move `StandardCharsets.UTF_8` to a property or import static?

Reply:
Not on my changes but OK :-)